target/linux/qualcommax/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -75,6 +76,11 @@ ipq807x_setup_macs()
 			done
 			[ "$(mtd_get_mac_ascii u_env eth2addr)" != "$label_mac" ] && wan_mac=$label_mac
 		;;
+		linksys,mx8500)
+			label_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
```

Comment:
This can probably be replaced by NVMEM instead as we can read ASCII MAC-s as well

Reply:
This is not possible, see the discussion here: https://github.com/openwrt/openwrt/pull/14051#discussion_r1443286488